compiler/z/codegen/BinaryEvaluator.cpp

```diff
@@ -87,9 +87,9 @@ extern void PRINT_ME(char * string, TR::Node * node, TR::CodeGenerator * cg);
  */
 TR::RegisterPair * lnegFor32Bit(TR::Node * node, TR::CodeGenerator * cg, TR::RegisterPair * targetRegisterPair, TR::RegisterDependencyConditions * dep = 0)
    {
+   TR::LabelSymbol * cFlowRegionStart = TR::LabelSymbol::create(cg->trHeapMemory(),cg);
```

Comment:
Similar comment as in https://github.com/eclipse/openj9/pull/3408#discussion_r227409032.

Reply:
Used `cFlowRegionStart` and `cFlowRegionEnd` consistently in 72fbbb2 and replaced `TR::LabelSymbol::create` with `generateLabelSymbol` in 47fc184.